tasks/create-pyxis-image/tests/test-create-pyxis-image-multiple-containerimages.yaml

```diff
@@ -82,5 +85,32 @@ spec:
                 cat $(workspaces.data.path)/mock_create_container_image.txt
                 exit 1
               fi
+
+              if [ $(cat $(workspaces.data.path)/mock_skopeo.txt | wc -l) != 6 ]; then
```

Comment:
Isn't this already checked (but more thoroughly) by the code below?

Reply:
yeah, ok, removed